src/plugins/createAKS/createAKSClusterPlugin.ts

```diff
@@ -0,0 +1,62 @@
+import * as vscode from "vscode";
+import { ILocalPluginHandler, LocalPluginArgs, LocalPluginEntry, LocalPluginManifest, ResponseForLanguageModelExtended } from "../../types/@azure/AzureAgent";
+import { getReadySessionProvider } from "../../auth/azureAuth";
+import { failed } from "../../commands/utils/errorable";
+import { getAKSClusterPluginResponse } from "../shared/pluginResponses";
+
+const createAKSClusterFunctionName = "create_aks_cluster";
+export const createAKSClusterPluginManifest: LocalPluginManifest = {
```

Comment:
💡 only fyi @hsubramanianaks + @kejatura-dev - I take you both have discussed this implementation. Just a note. aka `LocalPluginManifest` type usage here.

Reply:
Yes. This is a required step in enabling plugin with GH Copilot for Azure (@azure)